scripts/apps/authoring-react/authoring-section/authoring-section.tsx

```diff
@@ -90,7 +95,7 @@ export class AuthoringSection<T> extends React.PureComponent<IPropsAuthoringSect
                     backgroundColor: themeApplies ? this.props.uiTheme.backgroundColor : undefined,
                     display: 'flex',
                     flexDirection: 'column',
-                    gap: '12px',
+                    gap: this.props.useHeaderLayout === true ? '12px' : '32px',
```

Comment:
this would be better handled via `gap-` classes

Reply:
I've replaced the hardcoded values with css same variables that are used in those CSS classes.